src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -394,7 +468,7 @@ export class ChallengeModal extends Modal<Events, ChallengeModalProperties, any>
         let player_id = 0;
         if (this.props.mode === "player") {
             player_id = this.props.playerId;
-            if (!player_id || (player_id === data.get("user").id)) {
+            if (!player_id || player_id === data.get("user").id) {
```

Comment:
Things like this worry me. Clearly the author of this line thought order of operations are ambiguous (at least to a human), and the parentheses make it explicit.

Luckily it appears they keep parens for commonly confusing orders (`&&` mixed with `||`) https://github.com/prettier/prettier/issues/187

Reply:
yeah... new way looks tidier but you need to work harder to make sure you're reading it right, which goes against best coding practice of readability

Reply:
And to be fair, as with `return (null)`  , the extra parens here create cognitive load, in the form of "wait, why did that put that there, am I reading it right?" so I support prettier here.